web/api/v1/api.go

```diff
@@ -56,10 +63,10 @@ type response struct {
 
 // Enables cross-site script calls.
 func setCORS(w http.ResponseWriter) {
-	w.Header().Set("Access-Control-Allow-Headers", "Accept, Authorization, Content-Type, Origin")
-	w.Header().Set("Access-Control-Allow-Methods", "GET")
-	w.Header().Set("Access-Control-Allow-Origin", "*")
-	w.Header().Set("Access-Control-Expose-Headers", "Date")
+	w.Header().Set("Access-Control-Allow-Headers", corsAllowedHeaders)
```

Comment:
TBH, I don't quite get the reason for making the values const and then inlining the keys.
There's reuse in the tests, which guards against typos – but only for the values.

I would just make it a global `var corsHeaders map[string]string`. `setCors()` and the test can then simply use a loop over that.